src/game.h

```diff
@@ -518,6 +524,8 @@ class Game
 
 	WildcardTreeNode wildcardTree{false};
 
+	std::map<uint32_t, std::shared_ptr<House>> houses;
```

Comment:
A `flat_map` would shine here: every house is inserted on startup, and then it's only ever read